ui/app/AppLayouts/Wallet/controls/StatusNetworkSelector.qml

```diff
@@ -0,0 +1,224 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14 as QC
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Utils 0.1
+
+import utils 1.0
+
+/*!
+   \qmltype StatusNetworkSelector
```

Comment:
What is the difference between this `StatusNetworkSelector` and the existing `NetworkSelector`? What stops us from keeping only one around?